### DISCUSSION TITLE
Luckfox forum

### DISCUSSION THREAD

**Discussion by freetoair** [LuckfoxTECH/luckfox-pico#171] [General]
Unfortunately I have to note that the Luckfox forum is totally dysfunctional. The security policy on the forum is burdensome and slow. Just for the approval to post the topic, it takes several days. You should simply avoid the Luckfox forum!

That's why I'm asking a question from the forum here:

I would like to know, when a project is finished for any environment (Ubuntu,buildroot), which files need to be archived so that the project can be started from the same position later? Is only kernel .config buildroot.config and of course .dts enough or are there some other files that need to be saved?

And I can now confirm that ESP-hosted has been successfully ported to Luckfox-pico. It works very well !

**Comment by luckfox-eng33** [comment]
Sorry for the inconvenience, but due to China's National Day, our staff was on vacation and we forgot to notify forum users separately.
The security policy is a last resort because of the frequent posting of anachronistic content by users who have no choice but to .

As for your question, I'm not sure what you're changing, but in general we can manage projects directly through git.
Individual files may require other operations, depending on the specific location of the modifications
